services/dashboard/dashboard.js

```diff
@@ -66,6 +65,27 @@ angular.module('dashboard', [
     restapiRoot: location.origin + location.pathname,
     restapiQueryInterval: 3 * 1000, // in milliseconds
     restapiQueryTimeout: 30 * 1000, // in milliseconds
-    restapiTaskLevelMetricsQueryLimit: 100
-  })
-;
\ No newline at end of file
+    restapiTaskLevelMetricsQueryLimit: 100,
+    loginUrl: location.origin + location.pathname + 'login/login.html'
+  });
+
+app.factory('authInterceptor', ['$rootScope', '$q', '$window', 'conf', function ($rootScope, $q, $window, conf) {
```

Comment:
Create a new js file `authentication.js` under service directory and move new code to it. Maybe you can use our chained declaration without writting `app.` to a factory or a config.